tests/v2/validation/provisioning/hosted/hosted_provisioning_test.go

```diff
@@ -1,204 +0,0 @@
-package provisioning
```

Comment:
I think your rebase is off, this shouldn't show up as a commit, I'm confused this is here.

Reply:
should be resolved now